examples/elixir/tcp_inlet_and_outlet/02-inlet.exs

```diff
@@ -0,0 +1,10 @@
+["setup.exs"] |> Enum.map(&Code.require_file/1)
+
+# Parse argument.   Usage:  elixir 02-inlet.exs inlet_listen_port
+[lport_s] = System.argv
+{lport, ""} = Integer.parse(lport_s)
+
+{:ok, _pid} = Ockam.Transport.Portal.InletListener.start_link(port: lport, peer_route: [Ockam.Transport.TCPAddress.new("localhost", 4000), "outlet"])
+Ockam.Transport.TCP.start()
```

Comment:
You are right!.   It worked when testing just because it took me some time to _connect_  a socket to the inlet side after starting the script,   so by the time the InletListener do receive a connection and attempts to handshake,  the TCP() transport is already up. 

Will fix

Reply:
This should be also addressed in other examples